internal/config/testdata/v11.xml

```diff
@@ -0,0 +1,12 @@
+<configuration version="10">
```

Comment:
Not actually a v11 config, not that there are any changes in here anyway... ;) Maybe we should include some options in our example configs to make sure they parse correctly.